server/api/student/student.model.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var schoolYearSchema = new Schema({
+	schoolYear: Number,
+	allAbsences: Number,
+	tardy: Number,
+	present: Number,
+	enrolled: Number
+});
```

Comment:
Indented with tabs.

We haven't discussed any sort of style guide for this project. Since the majority of the boilerplate code is it is indented with spaces we could stick with that. What do you think?